WordPress/src/main/java/org/wordpress/android/ui/media/MediaBrowserActivity.java

```diff
@@ -900,7 +901,7 @@ private void addMediaToUploadService(@NonNull MediaModel media) {
         // Start the upload service if it's not started and fill the media queue
         if (!NetworkUtils.isNetworkAvailable(this)) {
             AppLog.v(AppLog.T.MEDIA, "Unable to start UploadService, internet connection required.");
-            return;
+            ToastUtils.showToast(this, R.string.no_network_message, ToastUtils.Duration.SHORT);
```

Comment:
thanks for having an eye for this - no this was totally accidentaly, got it back in 1ff3e23